client/src/app/shared/components/motion-poll-detail-content/motion-poll-detail-content.component.html

```diff
@@ -43,7 +43,7 @@
         </table>
 
         <!-- Chart -->
-        <div class="doughnut-chart">
+        <div *ngIf="!poll.hasSpecialVoteValues" class="doughnut-chart">
```

Comment:
This line does not work on motion poll slide (chart is always visible). - I have to merge this PR now (rest works). We can fix it later in a new PR.